### ISSUE TITLE
Non-zero exit code still results in success

### ISSUE THREAD

**Issue by bwolfe-fermata** [Luka-J9/sbt-buf#25]
Great plugin - excited to be able to easily run buf commands in CI without external dependencies!

Running `buf lint` from the command line produces the expected non-zero exit code:
```
buf lint
example.proto:1:1: : example error from linter
➜  development/ echo $?
100
```

Running `sbt bufLint` results in :

```
sbt bufLint
[info] example.proto:1:1 : example error from linter
[success] Total time: 0 s
```

It seems the `bufLint` task still reports as successful, so our CI builds still pass even with these failures.
Other tasks seem to behave the same, including the 'passthrough' such as `buf format --diff --exit-code;` - which should fail, but seems to ignore the error code.

Any thoughts?

**Comment by Luka-J9** [comment]
Hi! Thanks for the kind words!

Looks like it's a minor issue with the integration with the sbt auto plugin. Basically it should throw on a non 0 return for sbt to treat it as an error but it's just returning the value. It's a quick fix but I'm out of town this weekend, I can address and release a new version when I'm back though!

**Comment by bwolfe-fermata** [comment]
Thanks for such a prompt response. No rush, have a great weekend! It looks like the CI builds fail on that branch with the change to java env instead of java+scala (no sbt found).

**Comment by Luka-J9** [comment]
Yeah I was attempting to get the build going before leaving since the change was minor.

Building a release now - should be available shortly under 0.0.5
